CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+  * a hash of the contents of this URL
+
+* DRep retirement certificate. Including:
+  * DRep ID
+  * epoch number after which the DRep will retire
+
+* Vote delegation certificate. Including:
+  * stake credential
+  * DRep ID
+
+// TODO: Provide CBOR specification in the annexe for those new certificates.
+
+The authorization scheme (i.e. which signatures are required) mimics the existing stake delegation certificate authorization scheme.
+
+#### New Stake Distribution for DReps
+
+In addition to the existing per-stake-credential distribution and the per-stake-pool distribution, we will now have a new per-DRep stake distribution.
+
+This distribution will determine how much stake is backed by each `yes` (or `no`) vote from a DRep. The exact stake snapshot used for block production by the SPOs will also be used for the DReps for voting.
+
+#### Definitions surrounding Voting Stake
+
+We define some notions around voting stake:
+
+* Lovelace contained in a transaction output is considered **active for voting** if:
+  * It contains a registered stake credential.
+  * The stake credential has delegated its voting rights to a registered DRep.
+* Relative to some given percentage `P`, we say that there is **sufficient active voting stake** if
+  the ratio of the *total active voting stake* to the *total stake in circulation* is at least `P`.
+* Relative to a percentage `P`, a DRep (SPO) **vote threshold has been met** if the sum of the relative stake controlled by the DReps (SPOs) voting 'yes' to a governance action minus the sum of the relative stake controlled by the DReps (SPOs) voting 'no' is at least `P`.
+
+> **Note**
+> There are several alternative definitions for "the total stake in circulation":
+>  1. The sum of the UTxO and the rewards accounts.
+>  2. The sum of the UTxO, the rewards accounts, the fee pot, and the deposit pot.
+>  3. The total ADA supply (i.e. 45 billion ADA) minus the reserves[^3].
+>  We leave the choice open for discussion.
+
+## Rationale
+
+### Role of the Constitutional Committee
+
+At first, the Constitutional Committee may sound like a special committee that's granted extra power over DReps. However, because DReps can replace a committee at any time and DReps votes are also required to ratify every governance action, the Constitutional Committee has no more (and may, in fact, have less) power than the DReps. Given this, what role does the committee play, and why isn't it superfluous? The answer is that the committee solves the bootstrapping problem of this new governance framework. Indeed, as soon as we pull the trigger and enable such a framework to become active on-chain, without a Constitutional Committee, there would rapidly need to be sufficient DReps, so that the system did not rely solely on SPO votes. We cannot yet predict how active the community will be in registering as DReps, nor how reactive other Ada holders will be regarding delegation of votes.
```

Comment:
I find this a little unconvincing. It's not clear to me why we couldn't start with e.g. the CF and IOG registering themselves as DReps and delegating their stake to themselves. They could still initiate HFs! Indeed, I would have guessed that the reason for omitting the AVST threshold for some actions is precisely so that you can take them before you have a fully active governance system (but I don't know - I'd love to be told!).

I think it might help to separate this section into an explanation of the "hard" effect of the CC ("it helps to bootstrap the governance  system") and the "soft" effect of the CC ("it can be a respectable body to help with governance"). At the moment it seems mostly soft?

A common justification for bicameral systems like this is that it's helpful to have two bodies of legislators who move at different speeds. Since re-electing a CC is a lot of work and somewhat slow, they act as a brake on change. If someone manages to trick a bunch of people into delegating all their stake to them as a DRep, they could have a window of being able to enact whatever policy they like. The CC acts as a brake on that, which seems good.

Reply:
> Indeed, I would have guessed that the reason for omitting the AVST threshold for some actions is precisely so that you can take them before you have a fully active governance system.

That is correct. Note that the CF and IOG do not have enough stake to meet any reasonable AVST. Since we are just counting 'yes' votes, I personally cannot see any AVST at or below 50%. Note also that SPOs can block hard forks without any of these rules! The can always opt to not upgrade, and have ultimate veto power.

> At the moment it seems mostly soft?

Their purpose is at least twofold (and we can make this clearer):

1) Bootstrap the system
2) Provide a mechanism for enforcing the constitution, for those aspects of the constitution that we are not currently able to encode in the protocol itself. (in a sense, this is also just "bootstrapping the system").

> they could have a window of being able to enact whatever policy they like. The CC acts as a brake on that, which seems good.

Exactly (though it's a tricameral model, if you count the SPOs). The constitution should mention such concerns, as explicitly as we are able.

Reply:
Okay, I'm now confused: if we could start off the system with just a couple of big DReps and only take actions without AVST... why do we need the CC for bootstrapping at all?

Reply:
Would you trust that the liquid democracy was legitimate enough to make huge decisions for the entire community, if, say, only 20% of the the active stake was registered to vote? I could be wrong, I'd love others to chime in here too, but I think we'd need something over 50%.

So the CC is there to protect against low voting registration. But! Also to act as guardians of the constitution.

Reply:
... isn't that what AVST is for? But then yes, I can see the bind: you use AVST to protect against low vote registration, but then you can't get started. It's not obvious to me that you need something as substantial as the CC to fix that though.

Reply:
Perhaps we do not need something as substantial as the CC. But the CC also bootstraps the issue of wanting to have a constitution that is not obviously entirely enforceable on day one.

Reply:
Just thinking out loud: you could also handle that by having some sufficiently "big" DReps that they can veto unconstitutional stuff... but it's likely that the members of the CC won't have enough stake between them to do that. So they need a special role to allow that.

My next thought was that maybe the CC could just have a _veto_... but in fact they can't pass governance actions by themselves, so effectively they do just have a veto.

Reply:
> but it's likely that the members of the CC won't have enough stake between them to do that. So they need a special role to allow that.

exactly.

> maybe the CC could just have a veto... but in fact they can't pass governance actions by themselves, so effectively they do just have a veto.

indeed, from the very beginning it was thought of as just a veto, but we soon realized that this no different than a vote.

Reply:
Hi Jared! Is the MBO mentioned in the CIP the same MBO as mentioned at CardanoMBO.org?

Reply:
yes @TokhunGuru !

Reply:
@JaredCorduan I'd like to find out more, especially as I'm reviewing this proposal here, but I see the form at https://cardanombo.org doesn't currently advance beyond the screen ending with Question 7 (the Next button is anchor but not clickable)... is there anything we can do to follow up?

Reply:
@rphair in early 2023, there will be a space for discussion and follow-up. In the meantime, have you seen the video on Voltaire & the MBO from ScotFest on youtube?

Reply:
We have begun a governance division within a fledgling Cardano Professional Society. I'm preferring the multi division professional society model instead of a centralized organization. We are coordinating on the decentralized Matrix network:

The Cardano Professional Society Space:
https://matrix.to/#/#Cardano_Professional_Society:matrix.org